### ISSUE TITLE
Prowlarr Synocommunity not starting

### ISSUE THREAD

**Issue by ptzulu** [SynoCommunity/spksrc#5542]
### Is this a new Bug?

- [X] I checkd that the bug hasn't been reported before

### Package Name

Prowlarr

### Package Version

20220513-2

### Device Model

DS213j

### Device Architecture

ARMv7

### Firmware Version

DSM 7.1.1-42962 Update 3

### What happened?

Hi there.

So, just to be clear out of the bat: NO DOCKER SUPPORTED on this device, since most try to spin that way....

Sonnar v3 does work, but neither radarr or prowlarr work... 
I'm more interested in prowlarr first...

Anyhelp would be appreciated, thank you!

### Reproduction steps

```bash
1.Install
2.Repair
3.Repair
4.Repair
5.Repa.........
```

### Install Log

```log
2022/12/29 17:38:49     install prowlarr 20220513-2 Begin preinst
2022/12/29 17:38:49     Begin reload_inst_variables
2022/12/29 17:38:49     End reload_inst_variables
2022/12/29 17:38:49     Begin initialize_variables
2022/12/29 17:38:49     End initialize_variables
2022/12/29 17:38:49     ===> Step preinst. USER=sc-prowlarr GROUP= SHARE_PATH=
2022/12/29 17:38:49     install prowlarr 20220513-2 End preinst ret=[0]
2022/12/29 17:38:52     install prowlarr 20220513-2 Begin /bin/mkdir -p /volume1/@appstore/prowlarr -m 755
2022/12/29 17:38:52     install prowlarr 20220513-2 End /bin/mkdir -p /volume1/@appstore/prowlarr -m 755 ret=[0]
2022/12/29 17:38:52     install prowlarr 20220513-2 Begin /bin/rm -rf /volume1/@appstore/prowlarr
2022/12/29 17:38:52     install prowlarr 20220513-2 End /bin/rm -rf /volume1/@appstore/prowlarr ret=[0]
2022/12/29 17:38:52     install prowlarr 20220513-2 Begin /bin/mv -f /volume1/@tmp/synopkg/install.h5riRa/package /volume1/@appstore/prowlarr
2022/12/29 17:38:52     install prowlarr 20220513-2 End /bin/mv -f /volume1/@tmp/synopkg/install.h5riRa/package /volume1/@appstore/prowlarr ret=[0]
2022/12/29 17:38:52     install prowlarr 20220513-2 Begin /bin/rm -rf /var/packages/prowlarr
2022/12/29 17:38:52     install prowlarr 20220513-2 End /bin/rm -rf /var/packages/prowlarr ret=[0]
2022/12/29 17:38:52     install prowlarr 20220513-2 Begin /bin/mkdir -p /var/packages/prowlarr -m 755
2022/12/29 17:38:52     install prowlarr 20220513-2 End /bin/mkdir -p /var/packages/prowlarr -m 755 ret=[0]
2022/12/29 17:38:52     install prowlarr 20220513-2 Begin /bin/touch /var/packages/prowlarr/installing
2022/12/29 17:38:52     install prowlarr 20220513-2 End /bin/touch /var/packages/prowlarr/installing ret=[0]
2022/12/29 17:38:52     install prowlarr 20220513-2 Begin /bin/mv -f /volume1/@tmp/synopkg/install.h5riRa/INFO /var/packages/prowlarr/INFO
2022/12/29 17:38:52     install prowlarr 20220513-2 End /bin/mv -f /volume1/@tmp/synopkg/install.h5riRa/INFO /var/packages/prowlarr/INFO ret=[0]
2022/12/29 17:38:52     install prowlarr 20220513-2 Begin /bin/rm -rf /var/packages/prowlarr/scripts
2022/12/29 17:38:52     install prowlarr 20220513-2 End /bin/rm -rf /var/packages/prowlarr/scripts ret=[0]
2022/12/29 17:38:52     install prowlarr 20220513-2 Begin /bin/mv -f /volume1/@tmp/synopkg/install.h5riRa/scripts /var/packages/prowlarr/scripts
2022/12/29 17:38:52     install prowlarr 20220513-2 End /bin/mv -f /volume1/@tmp/synopkg/install.h5riRa/scripts /var/packages/prowlarr/scripts ret=[0]
2022/12/29 17:38:52     install prowlarr 20220513-2 Begin /bin/rm -rf /var/packages/prowlarr/WIZARD_UIFILES
2022/12/29 17:38:52     install prowlarr 20220513-2 End /bin/rm -rf /var/packages/prowlarr/WIZARD_UIFILES ret=[0]
2022/12/29 17:38:52     install prowlarr 20220513-2 Begin /bin/mv -f /volume1/@tmp/synopkg/install.h5riRa/WIZARD_UIFILES /var/packages/prowlarr/WIZARD_UIFILES
2022/12/29 17:38:52     install prowlarr 20220513-2 End /bin/mv -f /volume1/@tmp/synopkg/install.h5riRa/WIZARD_UIFILES /var/packages/prowlarr/WIZARD_UIFILES ret=[0]
2022/12/29 17:38:52     install prowlarr 20220513-2 Begin /bin/rm -rf /var/packages/prowlarr/conf
2022/12/29 17:38:52     install prowlarr 20220513-2 End /bin/rm -rf /var/packages/prowlarr/conf ret=[0]
2022/12/29 17:38:52     install prowlarr 20220513-2 Begin /bin/mv -f /volume1/@tmp/synopkg/install.h5riRa/conf /var/packages/prowlarr/conf
2022/12/29 17:38:52     install prowlarr 20220513-2 End /bin/mv -f /volume1/@tmp/synopkg/install.h5riRa/conf /var/packages/prowlarr/conf ret=[0]
2022/12/29 17:39:04     install prowlarr 20220513-2 Begin postinst
2022/12/29 17:39:04     Begin reload_inst_variables
2022/12/29 17:39:04     End reload_inst_variables
2022/12/29 17:39:04     Begin initialize_variables
2022/12/29 17:39:04     End initialize_variables
2022/12/29 17:39:04     ===> Step postinst. USER=sc-prowlarr GROUP= SHARE_PATH=
2022/12/29 17:39:04     Begin save_wizard_variables
2022/12/29 17:39:04     End save_wizard_variables
2022/12/29 17:39:04     Begin syno_sync_var_folder
2022/12/29 17:39:04     End syno_sync_var_folder
2022/12/29 17:39:04     Begin service_postinst
2022/12/29 17:39:04     End service_postinst
2022/12/29 17:39:04     install prowlarr 20220513-2 End postinst ret=[0]
2022/12/29 17:39:07     install prowlarr 20220513-2 Begin start-stop-status start
/var/packages/prowlarr/scripts/start-stop-status: line 115:  2307 Illegal instruction     (core dumped) ${SERVICE_COMMAND} >> ${OUT} 2>&1
2022/12/29 17:39:27     install prowlarr 20220513-2 End start-stop-status start ret=[1]
2022/12/29 17:39:43     (system) trigger prowlarr 20220513-2 Begin start-stop-status stop
2022/12/29 17:39:43     (system) trigger prowlarr 20220513-2 End start-stop-status stop ret=[0]
2022/12/29 17:42:08     start prowlarr 20220513-2 Begin start-stop-status start
/var/packages/prowlarr/scripts/start-stop-status: line 115:  4453 Illegal instruction     (core dumped) ${SERVICE_COMMAND} >> ${OUT} 2>&1
2022/12/29 17:42:28     start prowlarr 20220513-2 End start-stop-status start ret=[1]
2022/12/29 17:42:33     (system) trigger prowlarr 20220513-2 Begin start-stop-status stop
2022/12/29 17:42:33     (system) trigger prowlarr 20220513-2 End start-stop-status stop ret=[0]
2022/12/29 17:46:28     start prowlarr 20220513-2 Begin start-stop-status start
/var/packages/prowlarr/scripts/start-stop-status: line 115:  6464 Illegal instruction     (core dumped) ${SERVICE_COMMAND} >> ${OUT} 2>&1
2022/12/29 17:46:49     start prowlarr 20220513-2 End start-stop-status start ret=[1]
2022/12/29 17:46:54     (system) trigger prowlarr 20220513-2 Begin start-stop-status stop
2022/12/29 17:46:54     (system) trigger prowlarr 20220513-2 End start-stop-status stop ret=[0]
2022/12/29 17:51:48     start prowlarr 20220513-2 Begin start-stop-status start
/var/packages/prowlarr/scripts/start-stop-status: line 115:  8942 Illegal instruction     (core dumped) ${SERVICE_COMMAND} >> ${OUT} 2>&1
2022/12/29 17:52:08     start prowlarr 20220513-2 End start-stop-status start ret=[1]
2022/12/29 17:52:15     (system) trigger prowlarr 20220513-2 Begin start-stop-status stop
2022/12/29 17:52:16     (system) trigger prowlarr 20220513-2 End start-stop-status stop ret=[0]
2022/12/29 17:59:39     start prowlarr 20220513-2 Begin start-stop-status start
/var/packages/prowlarr/scripts/start-stop-status: line 115: 13199 Illegal instruction     (core dumped) ${SERVICE_COMMAND} >> ${OUT} 2>&1
2022/12/29 17:59:59     start prowlarr 20220513-2 End start-stop-status start ret=[1]
2022/12/29 18:00:04     (system) trigger prowlarr 20220513-2 Begin start-stop-status stop
2022/12/29 18:00:04     (system) trigger prowlarr 20220513-2 End start-stop-status stop ret=[0]
2022/12/29 18:01:36     start prowlarr 20220513-2 Begin start-stop-status start
/var/packages/prowlarr/scripts/start-stop-status: line 115: 14292 Illegal instruction     (core dumped) ${SERVICE_COMMAND} >> ${OUT} 2>&1
2022/12/29 18:01:56     start prowlarr 20220513-2 End start-stop-status start ret=[1]
2022/12/29 18:02:00     (system) trigger prowlarr 20220513-2 Begin start-stop-status stop
2022/12/29 18:02:01     (system) trigger prowlarr 20220513-2 End start-stop-status stop ret=[0]
2022/12/29 18:11:41     start prowlarr 20220513-2 Begin start-stop-status start
/var/packages/prowlarr/scripts/start-stop-status: line 115: 22344 Illegal instruction     (core dumped) ${SERVICE_COMMAND} >> ${OUT} 2>&1
2022/12/29 18:12:01     start prowlarr 20220513-2 End start-stop-status start ret=[1]
2022/12/29 18:12:04     (system) trigger prowlarr 20220513-2 Begin start-stop-status stop
2022/12/29 18:12:04     (system) trigger prowlarr 20220513-2 End start-stop-status stop ret=[0]
2022/12/29 18:13:55     start prowlarr 20220513-2 Begin start-stop-status start
/var/packages/prowlarr/scripts/start-stop-status: line 115: 23326 Illegal instruction     (core dumped) ${SERVICE_COMMAND} >> ${OUT} 2>&1
2022/12/29 18:14:15     start prowlarr 20220513-2 End start-stop-status start ret=[1]
2022/12/29 18:14:18     (system) trigger prowlarr 20220513-2 Begin start-stop-status stop
2022/12/29 18:14:18     (system) trigger prowlarr 20220513-2 End start-stop-status stop ret=[0]
```

**Comment by mreid-tt** [comment]
I would look at the file: `/var/packages/prowlarr/scripts/start-stop-status` and examine line 115 to see what it is trying to do. You can maybe even try to run the command manually on the terminal to see if you can replicate the error. With that there can perhaps be a patch made for it.

**Comment by hgy59** [comment]
@ptzulu the CPU of DS213j (armada370) does not support dotnet (see logs `illegal instruction`) and does not support newer radarr and prowlarr packages.
We already published prowlarr 20221218-4 and removed the unsupported archs from it.

We have to remove the spk version 20220513-2 in the repository, as it pretends to be availabel for archs armada370, 
alpine, monaco and armadaxp.
